packages/flutter_tools/lib/src/commands/run.dart

```diff
@@ -157,6 +157,11 @@ class RunCommand extends RunCommandBase {
         hide: !verboseHelp,
         help: 'Specify the project root directory.',
       )
+      ..addOption(
```

Comment:
This change should have a test in [the "run" command testfile](https://github.com/flutter/flutter/blob/master/packages/flutter_tools/test/commands.shard/hermetic/run_test.dart), verifying that the value passed by the option is received by the fake process manager used in the test.